app/views/layout.erb

```diff
@@ -24,6 +24,15 @@
 
         </div>
     </header>
+    <% if flash[:booked] && !flash[:booked].empty? %>
```

Comment:
We thought it was for the reasons you said above. Not 100% sure we need the whole each loop though, we could do it like the saved one below, any thoughts?